# Generated by Django 4.2.5 on 2023-09-18 09:36

from django.db import migrations, models
import users.validators


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required field. Enter Surname First Name Patronymic', max_length=150, validators=[users.validators.CustomFIOValidator], verbose_name='Фамилия Имя Отчество'),
        ),
    ]
